packages/components/storybook/colors.stories.mdx

```diff
@@ -142,7 +142,19 @@ Our color palette is organized into semantic color tokens, which have specific m
     subtitle="uswdsSystemColor"
     colors={{
       YellowVivid80: colors.uswdsSystemColorYellowVivid80,
-      YellwoVivid20: colors.uswdsSystemColorYellowVivid20,
+      YellowVivid20: colors.uswdsSystemColorYellowVivid20,
+    }}
+  />
+</ColorPalette>
+
+### Gold
```

Comment:
I wonder if the "Gold" titled ColorItem within "Yellow" ColorPalette above should be pulled down into being paired with this one now that it's here?

Reply:
Got rid of the Gold section and moved the gold uswds color under Yellow after checking with Jessica.

![image](https://github.com/user-attachments/assets/af22f2c1-e3a0-4c45-89a9-62f202492a2b)